src/main/scala/chisel3/util/LTL.scala

```diff
@@ -0,0 +1,392 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.util.ltl
```

Comment:
This is very bikesheddy but I feel like this package should just be `chisel3.ltl` or perhaps `chisel3.formal`. What do people think @mwachs5, @azidar, @seldridge?

Reply:
`chisel3.ltl` sounds good to me!